les/handler.go

```diff
@@ -1055,7 +1055,7 @@ func (pm *ProtocolManager) handleMsg(p *peer) error {
 		p.Log().Trace("Received tx status response")
 		var resp struct {
 			ReqID, BV uint64
-			Status    []core.TxStatus
+			Status    []txStatus
```

Comment:
This is an odd one. Why change this? Is this a network protocol packet? If so, don't we have it declared already in `protocol.go`. If not, perhaps we should? Just wondering here.

Reply:
It is a protocol packet and it is declared in protocol.go. Before the cleanup there was no les.txStatus, just core.TxStatus which contained the position and error fields too. When les.txStatus was created, we forgot to replace this instance (another reason the whole thing wasn't working).